gateway/src/apicast/policy/token_introspection/token_introspection.lua

```diff
@@ -55,7 +66,8 @@ local function introspect_token(self, token)
 
   --- Parameters for the token introspection endpoint.
   -- https://tools.ietf.org/html/rfc7662#section-2.1
-  local res, err = self.http_client.post(self.introspection_url , { token = token, token_type_hint = 'access_token'})
+  local res, err = self.http_client.post{self.introspection_url , { token = token, token_type_hint = 'access_token'},
+    headers = {['User-Agent'] = user_agent(), ['Authorization'] = self.credential}}
```

Comment:
User-Agent does not change, so it can be passed when creating the HTTP Client.

Reply:
Fixed.